integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -95,7 +94,7 @@ def _serialize_items(self, channel_metadata_items):
             sort_keys=True
         ).encode('utf-8')
 
-    def _transmit_create(self, channel_metadata_item_map, content_updated_mapping):
+    def _transmit_create(self, channel_metadata_item_map):
```

Comment:
here is an example of an inconsistent method signature - update and delete just take transmission item maps. since we are creating the content transmission records we're about to send before we send them, we can pass an item map in here too.